### PR TITLE
[CI] Move to the new sdk-insertions workload.

### PR DESCRIPTION
**Pull request by mandel-macaque**
Use the new templates and sdk-insertion project to unify the vs load.

### CODE REVIEW

**Review by dalexsoto — APPROVED** [review]

**Review by rolfbjarne — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

File: tools/devops/automation/templates/release/vs-insertion-prep.yml [hunk 1]
```diff
@@ -68,6 +69,33 @@ stages:
...
+        mkdir $(Build.SourcesDirectory)/nugets-blob
+        cp $(Build.SourcesDirectory)/package/* $(Build.SourcesDirectory)/nugets-blob
+        cp $(Build.SourcesDirectory)/vs-msi-nugets/* $(Build.SourcesDirectory)/nugets-blob
+      displayName: "Copy content for the nugets blob."
+
+    - template: templates/common/upload-vs-insertion-artifacts.yml@sdk-insertions
+      parameters:
+        githubToken: $(GitHub.Token)
+        githubContext: $(NupkgCommitStatusName)
+        blobName: nuget-artifacts
```
Comment (pjcollins) [inline tools/devops/automation/templates/release/vs-insertion-prep.yml:82]: On the Android and MAUI side I'm also using ` $(NupkgCommitStatusName)` for the `blobName` parameter, should we do that here?

File: tools/devops/automation/templates/release/vs-insertion-prep.yml [hunk 2]
```diff
@@ -68,6 +69,33 @@ stages:
...
+        packagePrefix: xamarin-macios
+        artifactsPath: $(Build.SourcesDirectory)/nugets-blob
+
+    - template: templates/common/upload-vs-insertion-artifacts.yml@sdk-insertions
+      parameters:
+        githubToken: $(GitHub.Token)
+        githubContext: $(VSDropCommitStatusName)
+        blobName: vs-insertion
```
Comment (pjcollins) [inline tools/devops/automation/templates/release/vs-insertion-prep.yml:90]: Same as above, should we use  `$(VSDropCommitStatusName)` for `blobName` here as well?

File: tools/devops/automation/templates/release/vs-insertion-prep.yml (hunk 1 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/release/vs-insertion-prep.yml:82]: +1

File: tools/devops/automation/templates/release/vs-insertion-prep.yml (hunk 2 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/release/vs-insertion-prep.yml:90]: +1

File: tools/devops/automation/templates/release/vs-insertion-prep.yml (hunk 1 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/release/vs-insertion-prep.yml:82]:
```suggestion
        blobName: $(NupkgCommitStatusName)
```

File: tools/devops/automation/templates/release/vs-insertion-prep.yml (hunk 2 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/release/vs-insertion-prep.yml:90]:
```suggestion
        blobName: $(VSDropCommitStatusName)
```

**Review by pjcollins — APPROVED** [review]

**Review by chamons — APPROVED** [review]
🎉

**Pull request merged**